Source/drlg_l2.cpp

```diff
@@ -357,8 +357,13 @@ static void DRLG_L2PlaceRndSet(BYTE *miniset, int rndper)
 			}
 			kk = sw * sh + 2;
 			if (found == TRUE) {
+#ifdef _XBOX
+				for (yy = max(sy - sh, 0); yy < min(sy + 2 * sh, DMAXY) && found == TRUE; yy++) {
+					for (xx = max(sx - sw, 0); xx < min(sx + 2 * sw, DMAXX); xx++) {
+#else
 				for (yy = std::max(sy - sh, 0); yy < std::min(sy + 2 * sh, DMAXY) && found == TRUE; yy++) {

```

Comment:
@marty28 could you check if defining `#define NOMINMAX 1` allows you to use `std::min` in VS03?

Reply:
@AJenbo I found it works. I made a commit reverting to the STL. Thx https://github.com/marty28/devilutionX/commit/bef88e29616df745ffc564a4fc446a186bec3c1d

Reply:
excelent, that will gets us a bit closer :)